connector/components/synchronizer.py

```diff
@@ -101,6 +110,310 @@ class Exporter(AbstractComponent):
     _base_mapper_usage = 'export.mapper'
 
 
+class GenericExporter(AbstractComponent):
+    """ Generic Synchronizer for exporting data from Odoo to a backend """
+    _name = 'generic.exporter'
+    _inherit = 'base.exporter'
+    _default_binding_field = None
+
+    def __init__(self, working_context):
+        super(GenericExporter, self).__init__(working_context)
```

Comment:
```suggestion
        super().__init__(working_context)
```

Reply:
Indeed. This is a simple forwardport to keep existing commits.